tags.json

```
@@ -318,6 +318,17 @@
   "arrow-up-circle": [
     "direction"
   ],
+  "arrow-up-down": [
+    "bidirectional",
+    "direction",
+    "swap",
+    "switch",
+    "network",
+	"mobile data",
+	"internet",
+	"reorder",
+	"move"
```

```suggestion
    "mobile data",
    "internet",
    "reorder",
    "move"
```
I think `reorder` & `move` could also be added to `arrow-left-right`.